client/src/components/form/Form.tsx

```diff
@@ -363,32 +418,179 @@ export function FormDialog(props: FormDialogProps) {
+          setType={setType}
+          typeError={typeError}
+          setTypeError={setTypeError}
+          scale={scale}
+          setScale={setScale}
+          language={language}
+          setLanguage={setLanguage}
+          pages={pages}
+          setPages={setPages}
+          pageRanges={pageRanges}
+          setPageRanges={setPageRanges}
+        />
+
+
+        <DateRangePickerPresets
+          issuanceDate={issuanceDate}
+          setIssuanceDate={setIssuanceDate}
+          hasError={issuanceDateError}
+        />
+
+
+
+        <Divider />
+
+        <FormDocumentGeolocalization
+          isMapOpen={isMapOpen}
+          setIsMapOpen={setIsMapOpen}
+          showGeoInfo={showGeoInfo}
+          setShowGeoInfo={setShowGeoInfo}
+          docCoordinatesError={docCoordinatesError}
+          setDocCoordinatesError={setDocCoordinatesError}
+          drawing={drawing}
+          setDrawing={setDrawing}
+          hideMap={hideMap}
+          setHideMap={setHideMap}
+          user={props.user}
+        />
+
+        <Divider />
+
+        <FormDocumentDescription
+          description={description}
+          setDescription={setDescription}
+          descriptionError={descriptionError}
+          setDescriptionError={setDescriptionError}
+        />
+
+        <Divider />
+
+        <FormDocumentConnections
+          documents={props.documents}
+          documentsForDirect={documentsForDirect}
+          setDocumentsForDirect={setDocumentsForDirect}
+          documentsForCollateral={documentsForCollateral}
+          setDocumentsForCollateral={setDocumentsForCollateral}
+          documentsForProjection={documentsForProjection}
+          setDocumentsForProjection={setDocumentsForProjection}
+          documentsForUpdate={documentsForUpdate}
+          setDocumentsForUpdate={setDocumentsForUpdate}
+          showConnectionsInfo={showConnectionsInfo}
+          setShowConnectionsInfo={setShowConnectionsInfo}
+        />
+
+        <Divider />
+
+        <FileUpload saveFile={(list) => setFiles(list)} />
+
+        <Divider />
+
+        <div className="mt-8 flex flex-col-reverse sm:flex-row sm:space-x-4 sm:justify-end">
+          <Button
+            className="w-full sm:w-auto mt-4 sm:mt-0 secondary"
+            variant="light"
+            onClick={() => setIsOpen(false)}
+          >
+            Cancel
+          </Button>
+          <Button
+            className="w-full sm:w-auto primary"
+            onClick={e => handleSubmit(e)}
+          >
+            Submit
+          </Button>
+        </div>
+
+      </form>
+    )
+  }
+
+  return (
+    <>
+{props.user && props.user.role === Stakeholders.URBAN_PLANNER && (
+  <Button
+    className="w-full primary"
+    style={{ borderRadius: '0.3rem', width: '11rem' }}
+    onClick={() => { setIsOpen(true); clearForm() }}
+  >
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <RiAddLine style={{ marginRight: '0.5rem' }} />
+      Add new document
```

Comment:
how about putting the button beside logout button, then separate the header n the dashboard using a border?